src/main/java/com/scalar/cassy/CassyBackupScheduler.java

```diff
@@ -0,0 +1,112 @@
+package com.scalar.cassy;
+
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "cassy-schedule",
+    mixinStandardHelpOptions = true,
+    description = "calls Cassy's backup command",
+    subcommands = {ClusterSnapshot.class, NodeSnapshot.class, NodeIncremental.class})
+public class CassyBackupScheduler implements Runnable {
```

Comment:
I think we should put the scheduler code in a separate gradle project. 
Temporarily we can duplicate the grpc code in both projects but eventually, we will have to make the grpc code as a sub package so that it can be imported in both projects.